src/components/Collection/CollectionView.tsx

```diff
@@ -8,56 +8,102 @@ import { useVirtualizer } from '@tanstack/react-virtual';
 import cx from 'classnames';
 import { debounce } from 'lodash';
 
-import CardViewItem from '@/components/Collection/CardViewItem';
-import GridViewItem from '@/components/Collection/GridViewItem';
-import { useLazyGetGroupsQuery } from '@/core/rtkQuery/splitV3Api/collectionApi';
+import ListViewItem from '@/components/Collection/ListViewItem';
+import PosterViewItem from '@/components/Collection/PosterViewItem';
+import { useLazyGetGroupSeriesQuery, useLazyGetGroupsQuery } from '@/core/rtkQuery/splitV3Api/collectionApi';
+import { useGetSettingsQuery } from '@/core/rtkQuery/splitV3Api/settingsApi';
 import { useLazyGetGroupViewQuery } from '@/core/rtkQuery/splitV3Api/webuiApi';
+import { initialSettings } from '@/pages/settings/SettingsPage';
+
+import type { SeriesType } from '@/core/types/api/series';
 
 type Props = {
   mode: string;
   setGroupTotal: (total: number) => void;
+  setTimelineSeries: (series: SeriesType[]) => void;
+  type: 'collection' | 'group';
+  isSidebarOpen: boolean;
 };
 
-const pageSize = 50;
+const defaultPageSize = 50;
```

Comment:
Should we move this to settings as well, seeing that you added settings query in here anyways?

Reply:
The GroupView endpoint only allows 100 group IDs so it would have to be a setting with a max limit of 100.. and people will complain because 90% of the people who would change the setting would want to be infinite size. We can add that setting afterwards.

Reply:
Well on the other hand we have infinite load anyway so page size doesn't really matter.